aten/src/ATen/core/SingletonSymNodeImpl.h

```diff
@@ -171,6 +189,11 @@ class C10_API SingletonSymNodeImpl : public SymNodeImpl {
  private:
   int64_t val_;
   int64_t coeff_;
+  at::Tensor values_;
```

Comment:
dumb Q: why do we need to store values in addition to offsets?

Reply:
Oh these are the offsets! I've been meaning to update the naming for these fields, I know its super confusing rn lol

Reply:
I'm good with using a more general name since we don't specifically want to tie singletons -> offsets. In fact, even for NestedTensor, a singleton might be associated with `lengths` instead. I just got confused by the name values overlapping with NT's values lol

Reply:
Renamed to `data`